products/sle-micro/main.pm

```diff
@@ -58,7 +59,20 @@ if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
     set_var('MAINT_TEST_REPO', $repos);
 }
 
-return 1 if load_yaml_schedule;
+testapi::set_distribution(DistributionProvider->provide());
+
+# set failures
+#$testapi::distri->set_expected_serial_failures(create_list_of_serial_failures());
+#$testapi::distri->set_expected_autoinst_failures(create_list_of_autoinst_failures());
```

Comment:
@sofiasyria do you please remember, why did you disabled this? Can we enable it (we need it for kernel tests).